dm/tests/shardddl2/run.sh

```diff
@@ -485,18 +485,14 @@ function DM_DropAddColumn_CASE() {
 		"\"source 'mysql-replica-02' has no error\"" 1
 
 	# after we skip ADD COLUMN, we should fix the table structure
```

Comment:
`# after we skip ADD COLUMN, we should fix the table structure`

this comment is no need any more

Reply:
deleted